guides/common/modules/ref_predefined-roles-available-in-project.adoc

```diff
@@ -3,6 +3,11 @@
 
 The following table provides an overview of permissions that predefined roles in {Project} grant to a user.
 
+[NOTE]
+====
+Certain {Project} plugins create roles automatically.
+====
```

Comment:
IMHO this whole note that not add any information to readers. -> I'd drop it.

Reply:
Dropped.